docs/source/intro/installation.rst

```diff
@@ -40,6 +40,12 @@ To assume the role of ``postgres`` run as root:
 
    $ psql
 
+.. tip::
+
+   If you see an error message like ``psql: FATAL:  role "<role_name>" does not exist``, probably you don't have any role created yet.
+   You can run ``psql -l`` to check the exist roles and specify the correct one with the ``-U`` option.
```

Comment:
How does this part happen? Does postgres in Conda get installed without any roles?

Reply:
It happened when no role is specified, as the example you provided. 
This tip is for generic advance installation section, in `intro/installation.rst` not for conda. So it will happened and adding `-U` can help.

Reply:
I see. But is the message than not incorrect? It is not so much that no role has been created yet, but that no role was specified?

Reply:
The same goes for the second part of the tip about the database. By default it will connect to the `postgres` database. This should also always exist. Why would a user ever encounter the problem that this default database doesn't exist. How the hell did you encounter that?

Reply:
Because I use postgresql and I didn't run `sudo su - postgres`.

Reply:
> Because I use postgresql 

Meaning? That is indeed what this section is about.

> and I didn't run `sudo su - postgres`

That is exactly my point. It is _that_ that is the problem. The tip should say that if you encounter the mentioned error, make sure to become the `postgres` system user.

Reply:
Sorry, I mean use postgresql from conda.

Reply:
If user follow the https://aiida.readthedocs.io/projects/aiida-core/en/stable/intro/install_conda.html, then it will encounter quite some issue in https://aiida.readthedocs.io/projects/aiida-core/en/stable/intro/installation.html. This is my point.